Manifests/ManagedPreferencesApplications/us.zoom.config.plist

```diff
@@ -483,6 +598,402 @@
 			<key>pfm_type</key>
 			<string>boolean</string>
 		</dict>
+		<dict>
+			<key>pfm_app_min</key>
+			<string>5.11.3</string>
+			<key>pfm_default</key>
+			<true/>
+			<key>pfm_description</key>
+			<string>Enable auto updates through the client</string>
+			<key>pfm_documentation_url</key>
+			<string>https://support.zoom.us/hc/en-us/articles/10365151862285</string>
+			<key>pfm_name</key>
+			<string>AU2_EnableAutoUpdate</string>
+			<key>pfm_note</key>
+			<string>This must be enabled to use any of the associated AU2 options.</string>
+			<key>pfm_title</key>
+			<string>Enable Updates</string>
+			<key>pfm_type</key>
+			<string>boolean</string>
+		</dict>
+		<dict>
+			<key>pfm_app_min</key>
+			<string>5.11.3</string>
+			<key>pfm_default</key>
+			<false/>
+			<key>pfm_description</key>
+			<string>Set the cadence of updates applied to the desktop client</string>
+			<key>pfm_documentation_url</key>
+			<string>https://support.zoom.us/hc/en-us/articles/10365151862285</string>
+			<key>pfm_exclude</key>
+			<array>
+				<dict>
+					<key>pfm_target_conditions</key>
+					<array>
+						<dict>
+							<key>pfm_present</key>
```

Comment:
I realize this block was taken from preferences already existing in the manifest, but I think the two conditions contradict.
I would replace the two conditions:
```
<key>pfm_present</key>
<false/>
<key>pfm_range_list</key>
<array>
	<false/>
</array>
```
With this one:
```
<key>pfm_n_range_list</key>
<array>
	<true/>
</array>
```

Reply:
I'm happy to do this for other preferences where the `pfm_default` = `false`, but my hesitation with doing this with the AU2 settings is because `AU2_EnableAutoUpdate` defaults to `true`.  As a result, even if set to the way you propose it could produce a situation where even if it's not actually added to a profile it is still treated as valid because the default value is `true` if that makes sense.  So for those AU2 settings, I'd like to leave the `pfm_present` keys in place, just in case

Reply:
In this case, instead of using what I suggested we should just remove the presence test and keep the range list as it is, like so:
```
<key>pfm_target_conditions</key>
<array>
    <dict>
        <key>pfm_range_list</key>
        <array>
        	<false/>
        </array>
        <key>pfm_target</key>
        <string>AU2_EnableAutoUpdate</string>
    </dict>
</array>
```
This way, `AU2_SetUpdateChannel` and the like will only be excluded when `AU2_EnableAutoUpdate` is set to `false`. When the latter is not present, it is not set to false and therefore the former keys will be enabled.

I've tested it in both IPE and PC and it's working well.